### PR TITLE
 feedbacks: Condition for sending email

### PR DESCRIPTION
**Pull request by vineeth-vk11**

### CODE REVIEW

File: routes/dashboard/portals/feedbacks-24x7/index.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ let router = express.Router();
 let feedbacksModel = fq("schemas/feedbacks");
 let feedbacks = require("./../feedbacks");
 
-["/", "/step-1"].forEach(step => {
+["/", "/step-1"].forEach((step) => {
```
Comment (sohailrajdev97) [inline routes/dashboard/portals/feedbacks-24x7/index.js:9]: Was this change done by prettier automatically?

File: routes/dashboard/portals/feedbacks-24x7/index.js [hunk 2]
```diff
@@ -54,38 +54,41 @@ router.post("/step-4", function(req, res, next) {
...
           res.renderState("custom_errors", {
             redirect: "/dashboard",
             timeout: 5,
             supertitle: "Couldn't submit feedback",
             message: "Failure",
-            details: err
+            details: err,
+          });
+        }
+        if (sendMail) {
```
Comment (sohailrajdev97) [inline routes/dashboard/portals/feedbacks-24x7/index.js:70]: This condition will always be false and thus this is a code path which will never be executed. Having blocks of dead code is not a good practice. 

My recommendation: Have this block controlled by an environment variable / database config.

File: routes/dashboard/portals/feedbacks-24x7/index.js [hunk 3]
```diff
@@ -54,38 +54,41 @@ router.post("/step-4", function(req, res, next) {
...
-            feedback +
-            "</b></p>"
-        });
         res.renderState("custom_errors", {
           redirect: "/dashboard",
           timeout: 2,
           supertitle: "Submitted Feedback.",
           message: "Success",
-          details: "Your feedback was recorded. Thank you :). Redirecting"
+          details: "Your feedback was recorded. Thank you :). Redirecting",
```
Comment (sohailrajdev97) [inline routes/dashboard/portals/feedbacks-24x7/index.js:91]: Revert this change if prettier allows.

File: routes/dashboard/portals/feedbacks-24x7/index.js [hunk 4]
```diff
@@ -21,7 +21,7 @@ let errorHandler = function(req, res, message) {
     supertitle: ".",
     callback: "/",
     message: "Validation Error",
-    details: message
+    details: message,
```
Comment (sohailrajdev97) [inline routes/dashboard/portals/feedbacks-24x7/index.js:24]: Revert this change if prettier allows.

File: routes/dashboard/portals/feedbacks-24x7/index.js [hunk 5]
```diff
@@ -54,38 +54,41 @@ router.post("/step-4", function(req, res, next) {
         instructor: instructoremail, // Instructor's email
         type: "24x7", // 24x7 or midsem
         responses: feedback,
-        createdOn: Date.now()
+        createdOn: Date.now(),
```
Comment (sohailrajdev97) [inline routes/dashboard/portals/feedbacks-24x7/index.js:57]: Revert this change if prettier allows.

File: routes/dashboard/portals/feedbacks-24x7/index.js (hunk 1 above)
Comment (vineeth-vk11) [reply routes/dashboard/portals/feedbacks-24x7/index.js:9]: yeah

**Review by sohailrajdev97 — APPROVED** [review]
